lib/scripts/services/contextMenu.js

```diff
@@ -1,19 +1,57 @@
+function _checkIfWordCountNotOne(inputString){
+    var s = inputString;
+    s = s.replace(/(^\s*)|(\s*$)/gi,"");
+    s = s.replace(/[ ]{2,}/gi," ");
+    s = s.replace(/\n /,"\n");
+    if (s.split(' ').length != 1){
+        alert('Please select a single word only. Phrases such as: "' + inputString + '" aren\'t allowed');
+        return true;
+    }
+    return false;
+}
+
+function _currentURLIsBlacklistedOrMTWIsOff(documentURL) {
```

Comment:
I'm not sure this function is a good idea. Did I understand correctly that, in case MTW is deactivated, the context menus will still appear, but the user will be alerted if he tries to use them? Ideally we should make the disabled context menus disappear instead...